### ISSUE TITLE
Wifi not working

### ISSUE THREAD

**Issue by LeonMQ** [SystemCrafters/guix-installer#6]
Trying to install on my pc and can not get wifi to work.
My pc has Intel Wifi 6 AX200
I have tried releases:
202106290243
202106240555
202106122242
When going through the graphical installer at the internet stage it only tries wired (I can't get ethernet to my pc).
Going into the shell I tried to set wifi up there but was unsure how exactly.
lspci -k shows that the network controller is found and is using iwlwifi
ifconfig -a returns eth0 and lo and some detail about them
iwconfig returns "eth0 no wireless extension", and "lo no wireless extension"

**Comment by LeonMQ** [comment]
Ok, I tried Ubuntu and Linux Mint and the issues is the same. It appears this is unrelated to the guix installer, so I will close this.

**Comment by senkowo** [comment]
Setting up `wpa_supplicant` and `dhclient` failed for me, and running `rfkill list all` printed that the wifi card was soft blocked. I fixed this issue by running `rfkill unblock wifi`. I have a Realtek RTL8188CE wifi card.
I didn't wanna make a new issue describing my problem and how i fixed it, so I'll just leave a comment here in case anyone else has the same issue.
